day2/gift.py

```diff
@@ -0,0 +1,22 @@
+class Gift:
+    def calculation_of_wrapping_paper(self, dimensions):
+        total = 0
+        dimensions_new = []
+        for dimension in list(dimensions):
+            dimension = dimension.strip()
+            dimensions_new.append(dimension)
```

Comment:
Eu ainda não consegui entender o que que essa lógica aqui faz

```python
dimensions_new = []
for dimension in list(dimensions):
    dimension = dimension.strip()
    dimensions_new.append(dimension)
```

Ela remove os espaços em branco em cada resultado?

Se vc remover essa lógica fora, as coisas param de funcionar? Pq eu imagino continue funcionando mesmo sem isso. 🤔

Reply:
Não funciona  mas eu usei um list comprehension agora para melhorar isso

Reply:
Sei lá o que é isso, mas ok

Reply:
list comprehension = é um for em uma linha @ricardoruwer